atom/browser/web_dialog_helper.cc

```diff
@@ -26,6 +28,94 @@
 
 namespace {
 
+class FileSelectHelper : public base::RefCountedThreadSafe<
```

Comment:
Minor nit: this can be just `base::RefCounted`, I don't see this class being used across threads. Also its good practice to make destructor private for ref counted classes.